Makefile

```diff
@@ -52,5 +52,6 @@ clean:
 .PHONY: dev
 dev:
 	go get github.com/pilu/fresh
-	docker-compose up
+	docker-compose start
```

Comment:
@aslakknutsen if you by "start postgres" mean take the postgres _image_ and create a container from it, then yes we do start a container. Did you mean we don't create images maybe? The distinction is important and `up -d` along with `--force-recreate` and is usually the best way to come to a reproducible environment. One might even think of running `docker-compose rm` first to be sure the volumes are not being reused. For postgres we might eventually mount in a persistent volume.